models/game.js

```diff
@@ -21,7 +21,8 @@ const Game = new Schema({
 	customGame: Boolean,
 	unlisted: Boolean,
 	isVerifiedOnly: Boolean,
-	chats: Array
+	chats: Array,
+	guesses: Object
```

Comment:
It's better to be more specific than this in the schema if possible -- cf. game-summaries/index.js
```	
gameSetting: {
		rebalance6p: Boolean,
		rebalance7p: Boolean,
		rebalance9p: Boolean,
		rerebalance9p: Boolean,
		casualGame: Boolean,
		practiceGame: Boolean,
		unlisted: Boolean
	},
```

Reply:
Yeah although the structure of the object isn't constant, as in it maps usernames which could be anything to guesses, so I'm not sure how you would express that.

Reply:
For an object you can show it's schema just like normal, with a nested object. See https://github.com/cozuya/secret-hitler/blob/master/models/bias.js for an example